core/legacy/gather/gather-runner.js

```diff
@@ -424,37 +418,6 @@ class GatherRunner {
 
     const baseArtifacts = passContext.baseArtifacts;
 
-    // Fetch the manifest, if it exists.
-    try {
-      baseArtifacts.WebAppManifest = await WebAppManifest.getWebAppManifest(
-        passContext.driver.defaultSession, passContext.url);
-    } catch (err) {
-      log.error('GatherRunner WebAppManifest', err);
-      baseArtifacts.WebAppManifest = null;
-    }
-
-    try {
-      baseArtifacts.InstallabilityErrors = await InstallabilityErrors.getInstallabilityErrors(
-        passContext.driver.defaultSession);
-    } catch (err) {
-      log.error('GatherRunner InstallabilityErrors', err);
```

Comment:
relevant context #13312

I don't know if this is still needed but ... can we replicate this in the gatherer?